modules/cnv-accessing-vmi-ssh.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-accessing-vm-consoles.adoc
+
+[id="cnv-accessing-vmi-ssh_{context}"]
+= Accessing a virtual machine instance via SSH
+
+You can use SSH to access a virtual machine (VM) after you expose port
+22 on the VM.
```

Comment:
Naughty, wicked Zoot. 
You can probably ignore my previous comment then :)

Reply:
There are some instances of "VM" in the rest of the collection, but I can replace them here.

Reply:
I've replaced all of the instances of VM and VMI that I could find.